src/gui/sdlmain.cpp

```diff
@@ -1849,24 +1858,45 @@ uint8_t GFX_SetSize(const int width, const int height,
 					}
 					glAttachShader(sdl.opengl.program_object, vertexShader);
 					glAttachShader(sdl.opengl.program_object, fragmentShader);
+
 					// Link the program
 					glLinkProgram(sdl.opengl.program_object);
+
 					// Even if we *are* successful, we may delete the shader objects
 					glDeleteShader(vertexShader);
 					glDeleteShader(fragmentShader);
 
 					// Check the link status
-					GLint isProgramLinked;
-					glGetProgramiv(sdl.opengl.program_object, GL_LINK_STATUS, &isProgramLinked);
-					if (!isProgramLinked) {
-						GLint info_len = 0;
-						glGetProgramiv(sdl.opengl.program_object, GL_INFO_LOG_LENGTH, &info_len);
-
-						if (info_len > 1) {
-							std::vector<GLchar> info_log(info_len);
-							glGetProgramInfoLog(sdl.opengl.program_object, info_len, nullptr, info_log.data());
-							LOG_ERR("SDL:OPENGL: Error link program:\n %s", info_log.data());
+					GLint is_program_linked;
```

Comment:
Assign a default that assumes negative/failure (0, I guess?)

Reply:
Yeah, I'll set it to `{}`.